hstream/src/HStream/Server/Common.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards   #-}
+
+module HStream.Server.Common where
```

Comment:
What does this "common" module do?

Reply:
Because other apis might also need some common functions in HStream.Server.Handler.

Reply:
So, why not just import HStream.Server.Handler?

Reply:
refactor handler like this:
  - HStream.Server.Handler.QueryHandler
  - HStream.Server.Handler.Common

Reply:
There will be cycle dependency. Since handler will also import apis.